worlds/kh2/docs/en_Kingdom Hearts 2.md

```diff
@@ -63,6 +63,21 @@ For example, if you are fighting Roxas, receive Reflect Element, then die mid-fi
 - Customize the amount and level of progressive movement (Growth Abilities) you start with.
 - Customize start inventory, i.e., begin every run with certain items or spells of your choice.
 
+<h2 style="text-transform:none";>What are Lucky Emblems?:</h2>
+Lucky Emblems are items that are required to beat the game if your goal is "Lucky Emblem". <br>
+Think of these "McGuffins" as you need X amount of Proof of Nonexistences to open the final door .
+
+<h2 style="text-transform:none";>What is Hitlist/Bounties?:</h2>
+The hitlist goal sets items named "bounty" on selected later fights, and you need to collect the amount you chose in your yaml to win. <br>
```

Comment:
```suggestion
The Hitlist goal adds "bounty" items to select late-game fights, and you need to collect X amount of them to win.<br>
```
Fixed grammatical errors and reworded to be more similar to the Lucky Emblem description.

Reply:
I wonder if it should be late-game fights/locations or just locations since they dont have to be on fights